src/puppetlabs/trapperkeeper/services/metrics/metrics_service.clj

```diff
@@ -29,7 +29,10 @@
     (:registry
       (core/get-or-initialize! (get-in-config [:metrics] {})
           (tk-services/service-context this)
-          domain))))
+          domain)))
+
+  (initialize-registry-settings [this domain settings]
+    nil))
```

Comment:
throwing an exception seems more obvious, I'll do that. Do you think that needs a test?